packages/shared/providers.yaml

```diff
@@ -2048,7 +2048,7 @@ datadog:
         - dev-tools
     auth_mode: API_KEY
     proxy:
-        base_url: https://${connectionConfig.siteParameter}/api
+        base_url: https://api.${connectionConfig.siteParameter}/api
```

Comment:
are we sure it's not going to break existing integration?
other question is `(ap1\.)` in the regex correct or a typo?

Reply:
> other question is (ap1\.) in the regex correct or a typo?
Yes, that is correct. 
![image](https://github.com/user-attachments/assets/d95449bd-1eb2-436a-9d76-c6cf5a068ce4)

> are we sure it's not going to break existing integration?
`api` is always included in the base_url so we were incorrect in not including it. I tested with it with our current configuration and it still works as expected.